cluster/addons/calico-policy-controller/cni-network-configmap.yaml

```diff
@@ -0,0 +1,23 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
```

Comment:
If you're trying to load this via addon-manager, you might need to add cluster-service: true.  I'm not sure if you're still having trouble there (or if this will work!).